### PR TITLE
Fully functioning ASE pipeline using Rasqual

### PR DESCRIPTION
**Pull request by freerkvandijk**

### CODE REVIEW

File: compute5/ASE/protocols/ASEReadCounter.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+if java -Xmx8g -XX:ParallelGCThreads=2 -Djava.io.tmpdir=${TMPDIR} -jar ${EBROOTGATK}/GenomeAnalysisTK.jar \
+ -T ASEReadCounter \
+ -R ${onekgGenomeFasta} \
+ -o ${ASEReadCountsChrOutput} \
+ -I ${bam} \
+ -sites ${selectVariantsBiallelicSNPsVcf} \
+ -L ${selectVariantsBiallelicSNPsVcf} \
+ -U ALLOW_N_CIGAR_READS \
+ -minDepth 0 \
+ --minMappingQuality 10 \
```
Comment (npklein) [inline compute5/ASE/protocols/ASEReadCounter.sh]: put in parameters?

File: compute5/ASE/protocols/CreateCountsTable.sh [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+
+#Loop through samples in VCF file and check if count file exists for all
+echo "#####################################"
+echo "Checking if a GATK ASEReadsCounter file exists for all samples specified in VCF file"
+echo "#####################################"
+
+for ((i=9; i<${#SAMPLESVCF[*]}; i++))
+do
+  	SAMVCF=${SAMPLESVCF[i]}
```
Comment (npklein) [inline compute5/ASE/protocols/CreateCountsTable.sh]: wrong indent

File: compute5/ASE/protocols/pasteReads.sh [hunk 3]
```diff
@@ -13,24 +15,40 @@
 #string samtoolsVersion
 #string tabixVersion
 #string sampleNum
-#list readCountFileExon,readCountFileGene
+#string CHR
+#list sampleName
```
Comment (npklein) [inline compute5/ASE/protocols/pasteReads.sh]: with rest?

File: compute5/ASE/protocols/rasqualize.R [hunk 4]
```diff
@@ -9,14 +9,14 @@ Quantile <-
...
 function(Y,gcvec,PLOT=F){
-	n=195
-        bin=Quantile(gcvec,n);
-        x=sort(unlist(lapply(split(gcvec,bin),mean)))
-        S=apply(Y,2,function(y){unlist(lapply(split(y,bin),sum))[as.character(0:(n-1))]});
-        Fs=log(t(t(S)/apply(S,2,sum))/apply(S,1,sum)*sum(as.numeric(S))); # Produces table of NA's until as.Numeric is given
-        print(dim(Fs))
-	print(length(x))
-	Gs=apply(Fs,2,function(y){smooth.spline(x,y,spar=1)$y});
+        n=ncol(Y)
```
Comment (npklein) [inline compute5/ASE/protocols/rasqualize.R]: comment why ncol(Y)

File: compute5/ASE/protocols/ASEReadCounter.sh (hunk 1 above)
Comment (freerkvandijk) [inline compute5/ASE/protocols/ASEReadCounter.sh]: Both values are in the parameters.csv now

File: compute5/ASE/protocols/CreateCountsTable.sh (hunk 2 above)
Comment (freerkvandijk) [inline compute5/ASE/protocols/CreateCountsTable.sh]: What's wrong here?

File: compute5/ASE/protocols/pasteReads.sh (hunk 3 above)
Comment (freerkvandijk) [inline compute5/ASE/protocols/pasteReads.sh]: Now all four lists are in one single #list statement

File: compute5/ASE/protocols/rasqualize.R (hunk 4 above)
Comment (freerkvandijk) [inline compute5/ASE/protocols/rasqualize.R]: Comment added and all indentation in this Rscript fixed.

**Comment by freerkvandijk** [comment]
Hi Niek,

I fixed all issues, except the one with the indent. What's wrong there?

Cheers,

F.

**Pull request merged**